src/crh/pedersen/mod.rs

```diff
@@ -120,6 +120,46 @@ impl<C: ProjectiveCurve, W: Window> FixedLengthCRH for CRH<C, W> {
     }
 }
 
+impl<C: ProjectiveCurve, W: Window> TwoToOneCRH for PedersenCRH<C, W> {
+    const LEFT_INPUT_SIZE_BITS: usize = W::WINDOW_SIZE * W::NUM_WINDOWS / 2;
+    const RIGHT_INPUT_SIZE_BITS: usize = Self::LEFT_INPUT_SIZE_BITS;
+    type Output = C::Affine;
+    type Parameters = Parameters<C>;
+
+    fn setup_two_to_one_crh<R: Rng>(r: &mut R) -> Result<Self::Parameters, Error> {
+        <Self as CRH>::setup_crh(r)
+    }
+
+    /// A simple implementation method: just concat the left input and right input together
+    ///
+    /// `evaluate` requires that `left_input` and `right_input` are of equal length.
+    fn evaluate_two_to_one_hash(
+        parameters: &Self::Parameters,
+        left_input: &[u8],
+        right_input: &[u8],
+    ) -> Result<Self::Output, Error> {
+        assert_eq!(
+            left_input.len(),
+            right_input.len(),
+            "left and right input should be of equal length"
+        );
+        // check overflow
+        // The reason to feature gate this `debug_assert` is that debug assert
```

Comment:
This is not feature gated?

Reply:
oops, my bad. I forgot to remove this comment